Gemfile

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-ruby "2.7.3"
+ruby "3.0.3"
```

Comment:
What do you think about this?

```suggestion
ruby File.read(".ruby-version").chomp
```

Reply:
Is it worth doing `#strip` instead?

Reply:
Updated :+1:

Reply:
There's not leading whitespace in the file. So `chomp` seems to be more accurate for this case but it doesn't really matter. :shrug: